packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -20,6 +21,156 @@ import 'scroll_position.dart';
 import 'scroll_position_with_single_context.dart';
 import 'scrollable.dart';
 
+/// Class that supplies children for the [ListWheelScrollView].
+///
+/// Instead of providing a concrete list of children to [ListWheelScrollView],
+/// this class allows several methods to lazily provide children. This is more
+/// efficient when there are too many or infinite children, by bringing in
+/// only the children that are visible in viewport, and removing them when they
+/// are no longer visible.
+///
+/// See also:
+///  * [ListWheelChildListDelegate], a delegate that supplies children using an
+///    explicit list.
+///  * [ListWheelChildLoopingListDelegate], a delegate that supplies infinite
+///    children by looping an explicit list.
+///  * [ListWheelChildBuilderDelegate], a delegate that supplies children using
+///    a builder callback.
+abstract class ListWheelChildDelegate {
+  /// Return the child at the given index. If the child at the given
+  /// index does not exist, return null.
+  Widget build(BuildContext context, int index);
+
+  /// Returns an estimate of the number of children this delegate will build.
+  int get estimatedChildCount;
+
+  /// Returns the true index for a child built at a given index. Defaults to
+  /// the given index, however if the delegate is [ListWheelChildLoopingListDelegate],
+  /// this value is the index of the true element that the delegate is looping to.
+  int trueIndexOf(int index) => index;
+
+  /// Called to check whether this and the old delegate are actually 'different',
+  /// so that the caller can decide to rebuild or not.
+  bool shouldRebuild(covariant ListWheelChildDelegate oldDelegate);
+}
+
+/// Class that supplies children for the [ListWheelScrollView].
+///
+/// Instead of providing a concrete list of children to [ListWheelScrollView],
+/// this class lazily supplies children using an explicit list. This is more
+/// efficient when there are too many children in the list, by bringing in
+/// only the children that are visible in viewport, and removing them when they
+/// are no longer visible.
+///
+/// Children are available in the range [0, explicit_list.length - 1], outside
```

Comment:
There isn't a thing called `explicit_list`. I think you also used the word 'explicit list' to refer to 2 different things now.